internal/datacoord/channel_manager.go

```diff
@@ -863,3 +872,11 @@ func getReleaseOp(nodeID UniqueID, ch *channel) ChannelOpSet {
 	op.Add(nodeID, []*channel{ch})
 	return op
 }
+
+func (c *ChannelManager) IsSilent() bool {
+	if c.stateTimer.hasRunningTimers() {
+		return false
+	}
+	lastSilentDuration := time.Since(c.lastActiveTimestamp)
+	return lastSilentDuration >= ChannelBalanceSilentDuration
```

Comment:
`ChannelBalanceSilentDuration` is a configurable duration. So it's just a guess there may be not incoming change?